src/main/java/org/folio/rest/impl/AcquisitionsUnitAssignmentsHelper.java

```diff
@@ -0,0 +1,64 @@
+package org.folio.rest.impl;
+
+import static org.folio.orders.utils.HelperUtils.buildQuery;
+import static org.folio.orders.utils.HelperUtils.handleDeleteRequest;
+import static org.folio.orders.utils.HelperUtils.handleGetRequest;
+import static org.folio.orders.utils.HelperUtils.handlePutRequest;
+import static org.folio.orders.utils.ResourcePathResolver.*;
+
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+import org.folio.rest.jaxrs.model.AcquisitionsUnitAssignment;
+import org.folio.rest.jaxrs.model.AcquisitionsUnitAssignmentCollection;
+
+import io.vertx.core.Context;
+import io.vertx.core.json.JsonObject;
+import me.escoffier.vertx.completablefuture.VertxCompletableFuture;
+
+public class AcquisitionsUnitAssignmentsHelper extends AbstractHelper {
+
+  private static final String GET_UNIT_ASSIGNMENTS_BY_QUERY = resourcesPath(ACQUISITIONS_UNIT_ASSIGNMENTS) + "?offset=%s&limit=%s%s&lang=%s";
+
+  AcquisitionsUnitAssignmentsHelper(Map<String, String> okapiHeaders, Context ctx, String lang) {
+    super(okapiHeaders, ctx, lang);
+  }
+
+  CompletableFuture<AcquisitionsUnitAssignmentCollection> getAcquisitionsUnitAssignments(String query, int offset, int limit) {
+    CompletableFuture<AcquisitionsUnitAssignmentCollection> future = new VertxCompletableFuture<>(ctx);
+
+    try {
```

Comment:
[Optional]:
From my point of view this try-catch block is redundant because the only one source of exception here is HelperUtils.encodeQuery(String query, Logger logger) method. And this exception may be thrown only as a result of UnsupportedEncodingException due to unsupported encoding charset in the URLEncoder.encode(String s, String enc) method. But we hardcoded absolutely supported standard encoding UTF-8 so I don’t see the possibility of such exception.